### ISSUE TITLE
Fix invalid option message for false for better DX

### ISSUE THREAD

**Issue by p3k** [stylelint/stylelint#5091]
> Clearly describe the bug

Extending the standard config StyleLint gives me an error about the unit after a zero value. This cannot be undone disabling the corresponding rule because that causes another error.

> Which rule, if any, is the bug related to?

`length-zero-no-unit`

> What code is needed to reproduce the bug?

```css
.foobar {
  margin: 0px; /* Unexpected unit (length-zero-no-unit) */
}
```
> What stylelint configuration is needed to reproduce the bug?

```js
module.exports = {
  extends: "stylelint-config-standard",
  rules: {
    "length-zero-no-unit": false
  }
};
```

> Which version of stylelint are you using?

13.7.2

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint test.css --config stylelint.config.js`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

Being able to disable the rule and not get the complaint about the unit after a zero value, anymore.

> What actually happened (e.g. what warnings or errors did you get)?

```plain
Unexpected option value "false" for rule "length-zero-no-unit"
```

**Comment by m-allanson** [comment]
Thank you for the clear write up.

Use null to disable a rule. More details at  https://stylelint.io/user-guide/configure#rules

**Comment by p3k** [comment]
gnah now i feel stupid 😬

**Comment by m-allanson** [comment]
Don't worry I've made exactly the same mistake. It's a fairly common issue. 

Which makes me wonder whether there's been discussion about improving that error message. 

e.g. something like this:

```
Unexpected option value "false" for rule "length-zero-no-unit". Do you want to disable this rule? Use "null" instead of "false".
```

Would probably save a good bit of confusion.

**Comment by jeddy3** [comment]
> Which makes me wonder whether there's been discussion about improving that error message.

SGTM!
